explainer-server/app/views/explainEditor.scala.html

```diff
@@ -68,6 +66,24 @@
             }
         }
 
+        /*
+         * This function (and the accompanying handler) exists because there doesn't seem to exist a way
+         * to generate a checkbox using ScalaTags (ScalaJS's HTML generation library)
+         */
+        function installDisplayTypeTickBox(displayType){
+            console.log(displayType);
+            var checkedstring = displayType=='Expandable' ? ' checked ' : ''
+            $(".explainer-editor__displayType-inner").html('<input class="explainer-editor__displayType-checkbox" type="checkbox" name="displayType" value="long" '+checkedstring+'> Long explainer')

```

Comment:
I'm still kinda sad that we're writing HTML like this. Could we sit together and try find a way to write this with ScalaTags? (Where you'll probably just prove to me that it's impossible but at least I'll learn something)